stacks-common/src/address/mod.rs

```diff
@@ -31,6 +31,7 @@ use std::convert::TryFrom;
 
 pub mod b58;
 pub mod c32;
+pub mod c32_old;
```

Comment:
I think adding `#[cfg(test]` above this mod declaration would make sure the old c32 encoding logic only gets compiled when tests are run.

Reply:
It will also prevent the benchmark from compiling, but that's probably okay?

Reply:
Could you use `#[cfg(any(test, feature = "testing"))]` or some such?  Maybe there needs to be a benchmarking build flag set?  I agree with @pavitthrap -- this shouldn't get compiled into a release build.

Reply:
Just a `#[cfg(test)]` is probably the easiest here. The `testing` feature should only really be used if the testing infrastructure needs the decorated module across build components: it is set in `debug` builds, so the code would end up getting included in those, but not `release` builds. Breaking the benchmark from compiling is fine-- this repo pretty regularly breaks benchmark code, which I'm fine with: benchmarking code isn't currently meant to be run regularly, and it's usually easy to fix back up whenever someone wants to rerun benchmarks.